tests/eko/io/test_struct.py

```diff
@@ -204,10 +204,12 @@ def test_version(self, tmp_path: pathlib.Path, eko_factory: EKOFactory):
         eko_factory.cache = None
         assert eko.access.path is not None
         read_closed = EKO.read(eko.access.path, dest=tmp_path)
-        supported_versions = [13, 14, 15]
 
         version = parse(read_closed.metadata.version)
-        assert (version.major == 0 and version.minor in supported_versions) or (
+        assert (
+            version.major + version.minor + version.micro >= 0
+            and version.is_postrelease is False
```

Comment:
```suggestion
            and not version.is_postrelease
```
my personal opinion :upside_down_face: 3 characters against 8